src/sentry/incidents/models.py

```diff
@@ -179,7 +178,6 @@ def snuba_values(self):
 
 
 class IncidentActivityType(Enum):
-    CREATED = 0
```

Comment:
Are there any existing records in the database using these values that will be broken now?

Reply:
No, we flushed out the table and I've double checked - no existing records have this